python313Packages.snowplow-tracker: 1.0.4 -> 1.1.0

dotlambda opened the pull request:
Diff: https://github.com/snowplow/snowplow-python-tracker/compare/refs/tags/1.0.4...1.1.0

Changelog: https://github.com/snowplow/snowplow-python-tracker/blob/1.1.0/CHANGES.txt


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

closes https://github.com/NixOS/nixpkgs/pull/383039

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

dotlambda committed:
python313Packages.snowplow-tracker: 1.0.4 -> 1.1.0

Diff: https://github.com/snowplow/snowplow-python-tracker/compare/refs/tags/1.0.4...1.1.0

Changelog: https://github.com/snowplow/snowplow-python-tracker/blob/1.1.0/CHANGES.txt

ethancedwards8 commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 384667`

---
### `aarch64-darwin`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>python312Packages.dbt-snowflake</li>
    <li>python312Packages.dbt-snowflake.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>dbt (python312Packages.dbt-core)</li>
    <li>dbt.dist (python312Packages.dbt-core.dist)</li>
    <li>python312Packages.dbt-bigquery</li>
    <li>python312Packages.dbt-bigquery.dist</li>
    <li>python312Packages.dbt-postgres</li>
    <li>python312Packages.dbt-postgres.dist</li>
    <li>python312Packages.dbt-redshift</li>
    <li>python312Packages.dbt-redshift.dist</li>
    <li>python312Packages.snowplow-tracker</li>
    <li>python312Packages.snowplow-tracker.dist</li>
    <li>python313Packages.snowplow-tracker</li>
    <li>python313Packages.snowplow-tracker.dist</li>
  </ul>
</details>

ethancedwards8 commented:
```
>   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
       >   File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
       >   File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
       >   File "<frozen importlib._bootstrap_external>", line 999, in exec_module
       >   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
       >   File "/nix/store/5fla72dsv8ydmprrpbqx2j8gsimshnr2-python3.12-dbt-snowflake-1.9.0/lib/python3.12/site-packages/dbt/adapters/snowflake/__init__.py", line 5, in <module>
       >     from dbt.adapters.snowflake.impl import SnowflakeAdapter
       >   File "/nix/store/5fla72dsv8ydmprrpbqx2j8gsimshnr2-python3.12-dbt-snowflake-1.9.0/lib/python3.12/site-packages/dbt/adapters/snowflake/impl.py", line 59, in <module>
       >     class SnowflakeAdapter(SQLAdapter):
       >   File "/nix/store/5fla72dsv8ydmprrpbqx2j8gsimshnr2-python3.12-dbt-snowflake-1.9.0/lib/python3.12/site-packages/dbt/adapters/snowflake/impl.py", line 80, in SnowflakeAdapter
       >     Capability.MicrobatchConcurrency: CapabilitySupport(support=Support.Full),
       >     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
       > AttributeError: type object 'Capability' has no attribute 'MicrobatchConcurrency'
```

dotlambda commented:
> 2 packages failed to build

no new failures

dotlambda committed:
python312Packages.dbt-snowflake: 1.9.0 -> 1.9.1

Diff: https://github.com/dbt-labs/dbt-snowflake/compare/refs/tags/v1.9.0...v1.9.1

Changelog: https://github.com/dbt-labs/dbt-snowflake/blob/v1.9.1/CHANGELOG.md

dotlambda committed:
python312Packages.dbt-core: 1.9.1 -> 1.9.2

Diff: https://github.com/dbt-labs/dbt-core/compare/refs/tags/v1.9.1...v1.9.2

Changelog: https://github.com/dbt-labs/dbt-core/blob/v1.9.2/CHANGELOG.md

dotlambda committed:
python312Packages.dbt-adapters: 1.11.0 -> 1.14.0

Changelog: https://github.com/dbt-labs/dbt-adapters/blob/main/dbt-adapters/CHANGELOG.md

dotlambda commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 384667`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 16 packages built:</summary>
  <ul>
    <li>dbt (python312Packages.dbt-core)</li>
    <li>dbt.dist (python312Packages.dbt-core.dist)</li>
    <li>python312Packages.dbt-adapters</li>
    <li>python312Packages.dbt-adapters.dist</li>
    <li>python312Packages.dbt-bigquery</li>
    <li>python312Packages.dbt-bigquery.dist</li>
    <li>python312Packages.dbt-postgres</li>
    <li>python312Packages.dbt-postgres.dist</li>
    <li>python312Packages.dbt-redshift</li>
    <li>python312Packages.dbt-redshift.dist</li>
    <li>python312Packages.dbt-snowflake</li>
    <li>python312Packages.dbt-snowflake.dist</li>
    <li>python312Packages.snowplow-tracker</li>
    <li>python312Packages.snowplow-tracker.dist</li>
    <li>python313Packages.snowplow-tracker</li>
    <li>python313Packages.snowplow-tracker.dist</li>
  </ul>
</details>

SuperSandro2000 commented on pkgs/development/python-modules/dbt-adapters/default.nix:
```diff
@@ -3,33 +3,38 @@
-    repo = "dbt-adapters";
-    tag = "v${version}";
-    hash = "sha256-l2OwGbYFqUF6NOpIxmgyd7B3sCsZTCgu4noaSfwRFps=";
+  # missing tags on GitHub
```
https://github.com/dbt-labs/dbt-adapters/issues/853
